nengo_sphinx_theme/theme/unstable_warning.html

```diff
@@ -0,0 +1,14 @@
+<br>
```

Comment:
Using `br` for spacing when referring to positional things like this can be unpredictable. We should be using something like margin or padding.

Reply:
Removed the `br` in my fixup